src/sagas/fetch/index.js

```diff
@@ -18,4 +18,24 @@ export function* fetchSaga(
   return null
 }
 
+export function* fetchAnythingSaga(action: Object): Generator<any, any, any> {
```

Comment:
Why? It referenced only from root saga, don't think we should mess up sagas within ducks